### DISCUSSION TITLE
onNothingFound callback prop for Autocomplete

### DISCUSSION THREAD

**Discussion by iakovalgaev** [mantinedev/mantine#1143] [Feature requests]
when no item is found in the list, I want to draw a button in the right section to add it.
I don't want to do this with nothing Found prop. This is not suitable for my design right now.

**Comment by rtivital** [comment]
You provide data, value and onChange handler, you can determine if nothing found based on data and value props.

**Reply by iakovalgaev** [reply]
Yes, I can take value from on Change and make an additional pass through data. I'm so focused on performance that I don't even want to make an extra pass. I am sorry. Thanks !
